<?xml version="1.0"?>
<launch>

    <arg name="rviz" default="true" />
    <arg name="project" default="multi_sensor_mapping"/>

    <!-- Lidar IMU Mapper -->
    <node pkg="$(arg project)" type="ground_segmentation_viewer" name="ground_segmentation_viewer" output="screen" respawn="false">
        <param name="session_path" value="/home/hkw/DATA/Dataset/MappingBackpackV3/公元大厦/GongyuanBuildingParkingLot(20230317)-Refined-20230317"/>
        <param name="config_path" value="mapping_backpack_v3/patchwork_param.yaml"/>
        <param name="debug_mode" value="false"/>
        <param name="ds_size" value="0.01"/>
    </node>


    <!--- Rviz -->
    <group if="$(arg rviz)">
      <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find multi_sensor_mapping)/rviz/ground_segmentation_viewer.rviz" />
    </group>
</launch>
